tests/cbmc/proofs/s2n_align_to/s2n_align_to_harness.c

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+#include "api/s2n.h"
+#include "utils/s2n_safety.h"
+
+#include <assert.h>
+#include <cbmc_proof/proof_allocators.h>
+#include <cbmc_proof/make_common_datastructures.h>
+
+int s2n_align_to_harness()
+{
+    uint32_t initial;
+    uint32_t alignment;
+    /* Division and modulo are too slow in CBMC to perform all necessary checks,
+     * so relevant assertions that can't be used currently have been left in comments. */
```

Comment:
can we raise an issue, and link to it here?

Reply:
Absolutely, I'll let the CBMC team know on Monday!